crew

```diff
@@ -384,7 +384,7 @@ def whatprovides (pkgName)
       if packageList[/\.filelist$/]
         packageName = File.basename packageList, '.filelist'
         File.readlines(packageList).each do |line|
-          found = line[/#{Regexp.new(pkgName)}/]
+          found = line[/#{Regexp.new(pkgName)}/] if line.ascii_only?
```

Comment:
Does `line[/#{pkgName}/]` works? (double regexp?)

Reply:
It's been working for years up until the ca_certificates package.

Reply:
Shouldn't ruby not break on non-ascii characters?

Reply:
https://stackoverflow.com/a/28962987

Reply:
Tried that already.  Doesn't work.  It's fine to ignore non-ascii characters.  You can't search for them anyway.

Reply:
`found = line[/#{Regexp.new(PKG name)}/u]` maybe?

Reply:
That is exactly what I tried.  You are welcome to try it for yourself also.

Reply:
@supechicken: I simplified the regex search and it does work.